stowdir/.local/bin/git-bl

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o noclobber
+
+main() {
+    local -r remote="${GIT_B_REMOTE:-origin}"
+    local -r show_default_branch="${GIT_B_SHOW_DEFAULT_BRANCH:-true}"
+    local preview_branches=()
+
+    if [[ "$show_default_branch" = "true" ]]; then
+        set +o errexit
+        local default_branch
+        default_branch="$(git rev-parse --abbrev-ref "$remote/HEAD" 2>/dev/null)"
+        # shellcheck disable=SC2181
+        if [[ $? -eq 0 ]]; then
+            # This is the default branch's remote tracking branch
+            preview_branches+=("$default_branch")
+            default_branch="${default_branch#"$remote/"}"
+            preview_branches+=("$default_branch")
+        fi
+        set -o errexit
+    fi
+
+    local -r filter_branch_name="sed 's/^[* ]*//' | cut -d ' ' -f1 -"
+    local -r filter_branch="echo {} | $filter_branch_name | head -1"
+    local -r filter_branches="printf '%s\n' {+} | $filter_branch_name"
+
+    local -r fzf_args=(
+        --preview-label="[ Git Log ]"
+        # Show the git-log --graph for the selected branch, the default branch, and their
+        # respective remote tracking branches
+        # TODO: Only show the default branch if it is "close enough" to the selected branch.
+        # Otherwise, the selected branch won't be visible.
+        --preview="
+            {
+                $filter_branches
+                $filter_branches | xargs -I % git rev-parse --abbrev-ref --symbolic-full-name %@{upstream} 2>/dev/null || true
```

Comment:
`git-gl` should also get this feature. Show not only the selected branch, but any remote tracking branches of it too.